app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/model/TransportInformationViewModel.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsddsfrontend.controllers.model
+
+import uk.gov.hmrc.cdsimportsddsfrontend.domain.{ArrivalTransportMeans, BorderTransportMeans}
+
+case class TransportInformationViewModel(container: Option[String] = Some("0"),
```

Comment:
This is just an observation - when naming our view model members we tried to match them to the field label names rather than using the domain nomenclature. We felt it would be easier to relate the view model members back to the UI i that way

For example the member for "Nationality of means of transport at border" would have been something like `NationalityOfMeansOfTransportAtBorder` rather than `registrationNationalityCode`

Reply:
renamed some members to better reflect the form fields